doc/manual/command-ref/nix-prefetch-url.xml

```diff
@@ -31,7 +31,7 @@
 
 <para>The command <command>nix-prefetch-url</command> downloads the
 file referenced by the URL <replaceable>url</replaceable>, prints its
-cryptographic hash, and copies it into the Nix store.  The file name
+cryptographic hash in base32, and copies it into the Nix store.  The file name
```

Comment:
afaik it uses a non-standard base32 implementation, so it might be worth mentioning.